cirq/experiments/n_qubit_tomography.py

```diff
@@ -96,8 +102,8 @@ def _make_state_tomography_matrix(self) -> np.ndarray:
 
         # Unitary matrices of each rotation circuit.
         unitaries = np.array([
-            protocols.unitary(
-                protocols.resolve_parameters(self.rot_circuit, rots))
+            protocols.resolve_parameters(self.rot_circuit,
+                                         rots).unitary(qubit_order=qubits)
```

Comment:
aren't you supposed to do `qubit_order = cirq.QubitOrder.explicit(qubits)` or did we simplify this?

Reply:
I've checked and think the present code is OK for two reasons:
1. Type annotation is `qubit_order: 'cirq.QubitOrderOrList'` which is defined as `Union[qubit_order.QubitOrder, Iterable[raw_types.Qid]]` see ([here](https://github.com/quantumlib/Cirq/blob/9c0eb4c0ea81f9b197cd2bfa87110ce531f3b6f9/cirq/ops/qubit_order_or_list.py#L26)).
2. The first thing `Circuit.unitary` does with the argument is to call `ops.QubitOrder.as_qubit_order(qubit_order)` (see [here](https://github.com/quantumlib/Cirq/blob/8612933e2dc8f4db1392187a72a2ecd132f6cee8/cirq/circuits/circuit.py#L1409)) which if the argument is an iterable calls `QubitOrder.explicit` (see [here](https://github.com/quantumlib/Cirq/blob/9c0eb4c0ea81f9b197cd2bfa87110ce531f3b6f9/cirq/ops/qubit_order.py#L125)).